GroupGitHubExercise/CardDeck.cs

```diff
@@ -10,12 +10,12 @@ class CardDeck
         private int currentCard;
         private Random _rnd = new Random();
 
-        private readonly int NUMBER_OF_FACES = 11;
-        private readonly int NUMBER_OF_SUITS = 0;
+        private readonly int NUMBER_OF_FACES = 13;
+        private readonly int NUMBER_OF_SUITS = 4;
 
         public CardDeck()
         {
-            NUMBER_OF_SUITS = Card.Suits.Length;
+         //   NUMBER_OF_SUITS = Card.Suits.Length;
```

Comment:
The code NUMBER_OF_SUITS = Card.Suits.Length; was there to suggest the right way to fix this.
You kinda did the opposite.

Try again.